perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -3071,7 +3071,17 @@ private boolean isThisLightweightSynchronization(PerunSession sess, Group group)
 				}
 			} catch (CandidateNotExistsException e) {
 				log.warn("getGroupSubjects subjects returned login {}, but it cannot be obtained using getCandidate()", login);
-				skippedMembers.add("MemberEntry:[" + subject + "] was skipped because candidate can't be found by login:'" + login + "' in extSource " + membersSource);
+				//If member can't be find in the member's extSource (we are missing other attributes) we can try find him in the group
+				UserExtSource subjectUserExtSource = new UserExtSource(source, login);
```

Comment:
I may be wrong but when the source and the membersSource differ the source may not be saved with the rich member so you will not find him in the mapping structure (getCandidate set membersSource as the main UserExtSource and additionalUserExtSources are fetched from the membersSource and they do not have to contain the source). Better to create this UserExtSource with membersSource.

Reply:
I think you are right here. I don't think we should skip the source at all, but at this moment the process is as you are saying.